content/info/about.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "About CompilED"
+date: 2019-03-28
+type: "info"
+poster: "poster-about.jpg"
+socmediaimg: ""
+hiliteimg: ""
+poster_sourceurl: "https://www.instagram.com/p/BUEdfYAgeJP/"
+poster_source: "Zarina Mustapha"
+---
+
+CompilED is a collection of post by the software developers at Center for Teaching and Learning (CTL) at Columbia University. These views are rooted in our professional and personal experiences developing educational technology. These posts are directed at software developers and architects, UX/UI designers, and media/video producers of all flavors, but everyone is welcome.
```

Comment:
I think a small typo here: "CompilED is a collection of post" -> "CompilED is a collection of posts"

Reply:
Fixed